config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace-core/hypertrace_core_event_attributes.conf

```diff
@@ -107,7 +107,7 @@ commands = [
               fqn: Span.attributes.tags,
               key: spanTags,
               value_kind: TYPE_STRING_MAP,
-              display_name: Tags,
+              display_name: Attributes,
```

Comment:
This is going to create a lot of confusion once deployed. We should send a broader communication about this changes once its deployed.

Reply:
+1 

All will be confused. We are got used to with name `Tags`.

Reply:
Sure, will drop a message